tensorflow-notebook/Dockerfile

```diff
@@ -15,4 +15,3 @@ RUN mamba install --quiet --yes \
     mamba clean --all -f -y && \
     fix-permissions "${CONDA_DIR}" && \
     fix-permissions "/home/${NB_USER}"
```

Comment:
Actually, one final newline is still here, otherwise GitHub would display that it didn't find the newline.